README.md

```diff
@@ -94,15 +94,37 @@ export class MyComponent implements OnInit {
 1. each endpoint must have an `operationId` defined
 1. `in: header` definitions are ignored
 1. `get` and `delete` methods do not contain `body`
+1. swagger file should contain values for the keys `host` and `basePath` so that each generated service method 
+can contain a link to the swagger UI method reference, e.g. `http://example.com/swagger/swagger-ui.html#!/Order/Order`
 
 ## Development
 
+* at least node 8 is needed
+
 ### Docker image
 
 1. `docker build . -t swagger-angular-generator`
 1. `docker run -u $(id -u) -it -v "$PWD":/code swagger-angular-generator bash`
 1. `npm i`
 
+### Testing
+
+##### How the testing works:
```

Comment:
We do not need a colon here (not used above). H4 instead of H5?

Reply:
done